activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -36,11 +36,25 @@ def after_teardown # :nodoc:
       class_attribute :pre_loaded_fixtures, default: false
       class_attribute :lock_threads, default: true
       class_attribute :fixture_sets, default: {}
+      class_attribute :database_transactions_config, default: {}
 
       ActiveSupport.run_load_hooks(:active_record_fixtures, self)
     end
 
     module ClassMethods
+      # Enable or disable transactions per database. This overrides the default
+      # setting as defined by `use_transactional_tests`, which applies to all
+      # database connection pools not explicitly configured here.
+      # Passing `nil` will revert config to the default setting, as specfied by
+      # `use_transactional_tests`.
+      def set_database_transactions(name, enabled)
+        if enabled.nil?
+          self.database_transactions_config.delete(name)
```

Comment:
```suggestion
          self.database_transactions_config = self.database_transactions_config.except(name)
```

Can't mutate the existing hash, because it may be inherited from the parent class